### ISSUE TITLE
Start sinusbot error

### ISSUE THREAD

**Issue by alienigenagamer** [sync667/UseTS-AdminBOT#3]
2016/03/20 13:13:27 37d6dd79 8ccbf939 ERROR  Error running JS-Wrap: TypeError: Cannot access member 'split' of undefined
    at _script__se___dmin____js (<anonymous>:13:48)
    at ScriptRunner (<anonymous>:191:4)
    at <unknown>v

**Comment by sync667** [comment]
Please paste full log and your configuration.

**Comment by pix0wl** [comment]
Hi, I have a probably related error:

`2016-03-21T11:09:47+01:00 Error running JS-Wrap: TypeError: Cannot access member '0' of undefined at _script__se___dmin____js (<anonymous>:106:13) at ScriptRunner (<anonymous>:191:4) at <unknown>
`

My log & config:
https://i.imgur.com/sKiQfiq.png
http://i.imgur.com/AQhc8OO.png

Also for some reason the script is telling me my config is wrong. 
I've tried various values without success.
`2016-03-21T11:09:47+01:00 [UseTSAdminBOT] START - Error code: TypeError: 'split' is not a function
2016-03-21T11:09:47+01:00 [UseTSAdminBOT] Your configuration is wrong! Check if you typed correct values.`

The bot seems to not work for me so far.

Edit2: I guess it would have been better, if I had started a new issue. Sorry for that mess.
